doc/tasks/composer_normalize.md

```diff
@@ -0,0 +1,37 @@
+# Composer Normalize
+
+If you are using `composer`, you have probably modified the file `composer.json` at least once to keep things nice
+and tidy.
+
+This task is a wrapper around a composer plugin for tidying up the file `composer.json`.
+
+The default configuration looks like:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        composer_normalize:
+            indent_size: 4
+            indent_style: space
+            update_lock: true
```

Comment:
What do you think about using the same name as in `localheinz/composer-normalize`, that is, `no_update_lock` instead of `update_lock`?

For reference, see [`Localheinz\Composer\Normalize\Command\NormalizeCommand#L97`](https://github.com/localheinz/composer-normalize/blob/1.1.2/src/Command/NormalizeCommand.php#L97).

Reply:
I did this for a better understanding, configuration properties shouldn't be negated, it's easier for everyone I think.

Reply:
@localheinz Are you ok with this or you really want to change it ?

Reply:
I'dd prefer to stick with the options the package provides. That makes it easier for people who are using the tool already.

Reply:
Ok